steve/util.py

```diff
@@ -206,7 +371,17 @@ def get_project_config():
     :returns: config file
 
     """
-    return SteveConfig().load()
+    try:
+        return SteveConfig().load()
+    except ConfigNotFound as e:
+        try:
+            for cfg in [IniConfig, YamlConfig]:
+                if cfg == SteveConfig:
+                    continue  # already tried
+                return cfg().load()
+        except ConfigNotFound:
+            pass
+    raise e   # original raised error
```

Comment:
I don't understand what this is trying to do. What situation would `.load()` raise a `ConfigNotFound` error but it'd succeed if we try the various config file formats? 

Also, I think this `raise e` needs to be indented.